src/aleph/http/client.clj

```diff
@@ -809,7 +811,7 @@
                           "websocket-deflater"
                           WebSocketClientCompressionHandler/INSTANCE)))
             (#(when (some? proxy-options)
-               (let [proxy (proxy-handler (assoc proxy-options :ssl? ssl?))]
+               (let [proxy (proxy-handler (assoc proxy-options :ssl? ssl? :keep-alive? keep-alive?))]
```

Comment:
Sorry, I was not clear enough... I meant just `assoc` here `true` to prevent self-shooting

```clojure
(assoc proxy-options :ssl? ssl? :keep-alive? true)
```

In such a case even if I passed `:keep-alive?`with my `proxy-options` (we never validate keys anyway), it still will be set to `true` when passing to the handler's builder. In such a case we don't need `:keep-alive?` being mentioned as an option.